airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryUtils.java

```diff
@@ -178,26 +178,34 @@ public static Table createTable(final BigQuery bigquery, final String datasetNam
    */
   public static void createPartitionedTableIfNotExists(final BigQuery bigquery, final TableId tableId, final Schema schema) {
     try {
-      final var chunkingColumn = JavaBaseConstants.COLUMN_NAME_AB_EXTRACTED_AT;
-      final TimePartitioning partitioning = TimePartitioning.newBuilder(TimePartitioning.Type.DAY)
-          .setField(chunkingColumn)
```

Comment:
Will there be any side effect in performance by losing the time partition on `ab_extracted_at` ?

Reply:
_thinking loud_  Should it be reverse as-in clustering by genId and partitioning by `extracted_at`

Reply:
I could be convinced to leave this unchanged :P

if we ever want to do the `delete ... where generation_id < ?` thing, then partitioning on gen ID is how we support that cheaply

and my theory is that partitioning on extracted_at is redundant, since bigquery can just optimize within each partition anyway? but I'm not super familiar with this